### PR TITLE
Update data storage solutions docsv2

### PR DESCRIPTION
**Pull request by drelliche**
Note that the title of this module (in the metadata) has been changed to match the internal title of the module, "Types of Data Storage Models"

All connected feedback in the quick module feedback in REDCap has already been edited to reflect this change.

### CODE REVIEW

File: data_storage_models/data_storage_models.md [hunk 1]
```diff
@@ -22,33 +29,30 @@ After completion of this module, learners will be able to:
...
 
-@comment
+@depends_on_knowledge_available_in
 
-**Is this module right for me?** @long_description
+@end
 
-**Estimated time to completion:** @estimated_time
+@version_history 
+No previous versions.
```
Comment (rosemm) [inline data_storage_models/data_storage_models.md:45]: I think we must be missing content here. There should be a link to each minor update, right? So 1.0.x should be listed.

File: data_storage_models/data_storage_models.md [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
+docs_version: 2.0.0
 language: en
 narrator: UK English Female
-title: Types of Data Storage Solutions
+mode: Textbook
+title: Types of Data Storage Models
 comment:  This course will focus on different data storage solutions available to an end user and the unique characteristics of each type. This course will also cover how each storage type impacts one's access to data and computing capabilities.
 long_description: This module is for people interested in understanding the types of data storage solutions available to them at their institution and why they might want to store their files or perform certain computational tasks in each.
-@estimated_time: 30 minutes
+@estimated_time_in_minutes: 30
```
Comment (rosemm) [inline data_storage_models/data_storage_models.md:15]:
```suggestion
estimated_time_in_minutes: 30
```

**Review by rosemm — CHANGES_REQUESTED** [review]
I think this is missing the version history from 1.0.x (see https://chop365.sharepoint.com/:w:/r/teams/RSCH-Arcus/Shared%20Documents/Arcus%20Drive/Education/R25/Reconstructing%20Version%20Descriptions.docx?d=w400f6dc02dc745ad89fada5c0277ffdf&csf=1&web=1&e=75kLrP)

File: data_storage_models/data_storage_models.md (hunk 2 above)
Comment (drelliche) [reply data_storage_models/data_storage_models.md:15]: oops, good catch!

**Comment by drelliche** [comment]
@rosemm you are totally right, in my head I misread it as version 1.0.2 instead of 1.1.2 so I didn't even peek at the version history! Oops!

**Review by rosemm — APPROVED** [review]

**Pull request merged**